templates/app/app.hbs

```diff
@@ -48,13 +48,13 @@ app.use('/forest', (request, response, next) => {
 requireAll({
   dirname: path.join(__dirname, 'routes'),
   recursive: true,
-  resolve: Module => app.use('/forest', Module),
+  resolve: (Module) => app.use('/forest', Module),
```

Comment:
[side note] I'm not sure we are consistent in our own code! 😅 (lint rules differ: sometimes I have to remove the parenthesis)

Reply:
Really, are you sure about that? I don't know such scenarios.
I'd love to have the opportunity to make the rules more consistent, feel free to share those inconsistent cases.

Reply:
@arnaudbesnier Yes, sorry about being vague 😅 ! If you look at **Forestadmin** repository source code, you should see 300+ occurrences of `x => y` without parenthesis.

<img width="275" alt="Capture d’écran 2020-08-13 à 09 03 19" src="https://user-images.githubusercontent.com/1575946/90104037-d7b91600-dd43-11ea-9684-919259ee6246.png">

If you try to add a parenthesis, you should see this linter error:

<img width="796" alt="Capture d’écran 2020-08-13 à 09 04 56" src="https://user-images.githubusercontent.com/1575946/90104183-19e25780-dd44-11ea-841b-dcaeb77610c9.png">

Still, this rule differs in **Forestadmin server**.